source/ecs/entity.d

```diff
@@ -231,6 +236,71 @@ public:
 	}
 
 
+	/**
+	 * Generates a new entity and assigns the component to it. If the component
+	 *     has no ctor or a default ctor then only it's type can be passed.
+	 *
+	 * Examples:
+	 * --------------------
+	 * @Component struct Foo { int x; } // x gets default initialized to int.init
+	 * auto em = new EntityManager!size_t;
+	 * em.gen!Foo(); // generates a new entity with Foo.init values
+	 * --------------------
+	 *
+	 * Params: component = a valid component.
+	 *
+	 * Returns: a newly generated Entity!T.
```

Comment:
It's fine, but you don't need to seperate single lines on documentation

Reply:
It's just my style. I find it easier to read the written documentation this way.